.github/workflows/integration_tests.yaml

```diff
@@ -16,14 +16,15 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
 
       - uses: actions/setup-python@v4
         with:
           python-version: "3.9"
 
       - name: Install dependencies
         run: |
-          git clone --depth 1 https://github.com/fal-ai/fal
```

Comment:
did we originally do this to speed up CI? I wonder if we should out the explicit shallow-clone in the new addition, since it really won't matter in GitHub Actions I think

Reply:
@squat Do you mean that it was copied from some older version in the past? I see that it was introduced as a whole in https://github.com/fal-ai/fal/pull/22/files

Regarding the shallow clone, it doesn't matter since the repo is pretty small right now, but it will start to matter the bigger it gets. Maybe a premature optimization, but seems like there is no harm in just doing it right and keeping that flag since we are already here.